### PR TITLE
Add Player panel link for admins

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request

This adds links to OOC, Dead chat, and Say chat (when ghosting) as admin

## Why It's Good For The Game

Makes it easier to administrate.

## Changelog
:cl:
admin: adds links to OOC, Dead chat, and Say chat (when ghosting) as admin
/:cl:

### CODE REVIEW

File: code/game/say.dm [hunk 1]
```diff
@@ -67,8 +67,16 @@ GLOBAL_LIST_INIT(freqtospan, list(
...
 
-	return "[spanpart1][spanpart2][CMSG_FREQPART][languageicon][CMSG_JOBPART][namepart][endspanpart][messagepart]"
+	return "[spanpart1][spanpart2][CMSG_FREQPART][languageicon][CMSG_JOBPART][compose_name_href(namepart)][endspanpart][messagepart]"
 
+/**
+	Allows us to wrap the name for specific cases like AI tracking or observer tracking
+
+	Arguments
+	- name {string} the name of the mob to modify.
+*/
```
Comment (TiviPlus) [inline code/game/say.dm:77]:
```suggestion
/**
  *Allows us to wrap the name for specific cases like AI tracking or observer tracking
  *
  * *Arguments
  * * name {string} the name of the mob to modify.
*/
```

**Comment by TiviPlus** [comment]
Is it intended that non-admin ghost players can see the link but nothing happens when you click on it?

**Comment by psykzz** [comment]
It's not, i'll have a look

File: code/game/say.dm (hunk 1 above)
Comment (psykzz) [reply code/game/say.dm:77]: The extra * will actually break the markup in the end.

With how i have it now the arguments will parse into a list

**Comment by Mr-Worth** [comment]
I don't know how difficult this would be but it would be nice if the link color inherited the color of the type of speech it is referencing.

**Comment by Mr-Worth** [comment]
Also hivemind speech does not make a link

**Pull request merged**